### PR TITLE
Kraken: fix crash in route_schedule caused by forbidden_uri 

### PR DESCRIPTION
**Pull request by xlqian**
https://jira.canaltp.fr/browse/NAVITIAII-2341

### CODE REVIEW

**Comment by xlqian** [comment]
Jenkins, test this please!

**Review by TeXitoi — APPROVED** [review]

File: source/jormungandr/tests/departure_board_tests.py [hunk 1]
```diff
@@ -1088,3 +1088,11 @@ def test_departure_schedule_departures_date_time_frequency_base_schedule(self):
         assert departures[2]["stop_date_time"]["departure_date_time"] == '20160103T191000'
         assert departures[2]["stop_date_time"]["base_departure_date_time"] == '20160103T191000'
         assert departures[2]["stop_date_time"]["data_freshness"] == 'base_schedule'
+
+    def test_routes_schedule_with_invalid_forbidden_uri(self):
+        """
+        there is no reason to have forbidden_uri in route_scheduel, but it shouldn't crash...
```
Comment (pbougue) [inline source/jormungandr/tests/departure_board_tests.py]: typo: route_schedu`les`

**Review by pbougue — COMMENTED** [review]
Did you check that we don't have the same kind of unprotected calls somewhere else?

**Review by benoit-bst — APPROVED** [review]

**Review by pbougue — APPROVED** [review]

**Comment by kinnou02** [comment]
We should make these exception recoverable too.

**Pull request merged**